lib/voxpupuli/release/rake_tasks.rb

```diff
@@ -52,3 +52,116 @@
     fail "Unable to find a CHANGELOG.md entry for the #{v} release."
   end
 end
+
+desc "Prepare a release"
+task "release:prepare" do
+  v = Blacksmith::Modulefile.new.version
```

Comment:
This now looks unused

Reply:
It is only used for end-user convenience to output the CLI commands to create the release branch:

```
git checkout -b release-#{v}
git commit -m "Release #{v}"
```